src/types/generated/operations.ts

```diff
@@ -1,336 +1,93 @@
-import * as Types from "./types";
+import * as Types from './types';
 
-export type CurrentTokenOwnershipFieldsFragment = {
-  token_standard: string;
-  token_properties_mutated_v1?: any | null;
-  token_data_id: string;
-  table_type_v1?: string | null;
```

Comment:
Any reason this file just disappears?

Reply:
hmm wait, which file? or you mean the `CurrentTokenOwnershipFieldsFragment` type? I think it got re-formatted